enzy_htp/mutation/api.py

```diff
@@ -367,8 +369,59 @@ def mutate_stru_with_tleap(
 
     return stru_cpy
 
+def mutate_stru_with_pymol(
+        stru: Structure,
+        mutant: List[Mutation],
+        in_place: bool = False,
+        ) -> Structure:
+    """mutate the {stru} to its {mutant} structure using PyMOL.
+    Args:
+        stru: the 'WT' structure
+        mutant: a list of Mutation() which describes a mutant to the 'WT'
+        in_place: if make the changes to the structure in-place 
+    Returns:
+        The mutated structure.
+    """
+
+    sp = PDBParser()
+    temp_file_name = fs.get_valid_temp_name("pymol_mutation_result.pdb")
+
+    # san check
+    for mut in mutant:
+        if not isinstance(mut, Mutation):
+            _LOGGER.error(
+                f"mutant takes only a list of Mutation(). Current mutant is: {mutant}")
+            raise TypeError
+
+    stru_cpy = copy.deepcopy(stru)
+
+    # 1. load stru into pymol
+    pi = interface.pymol
+    temp_session = pi.new_pymol_session()
+    pymol_obj_name, session = pi.load_enzy_htp_stru(stru_cpy, temp_session)
```

Comment:
We dont need to have a new name for the `temp_session` here since load_enzy_htp_stru() just return the reference of the input temp_session. (probably not a good design lol)